mmv1/main.go

```diff
@@ -56,6 +56,8 @@ var yamlTempMode = flag.Bool("yaml-temp", false, "copy text over from ruby yaml
 var handwrittenTempFiles = flag.String("handwritten-temp", "", "copy specific handwritten files over from .erb to go .tmpl.temp comma separated")
 var templateTempFiles = flag.String("template-temp", "", "copy specific templates over from .erb to go .tmpl.temp comma separated")
 
+var showImportDiffs = flag.Bool("show-import-diffs", false, "write go import diffs to stdout")
```

Comment:
Do you plan to add `show-import-diffs` flag to the CLI in the make file?
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/GNUmakefile#L61-L68

Reply:
No, for a couple of reasons:

- It dumps a **lot** of info into the console (and it's info that most developers don't care about)
- It runs a lot slower (since it has to run `goimports` twice)

I added the flag to make it easy for the core team to add future import fixes, and temporarily adding it to the makefile is how I found the `resource.go.tmpl` imports :-)

Reply:
Sounds good.